rites/roman1969/src/types/liturgical-day.ts

```diff
@@ -221,6 +221,13 @@ export type RomcalCalendarMetadata = {
    * The last day of the current liturgical season.
    */
   endOfSeason: string;
+
+  /**
+   * Whether the Nativity of the Lord occurs on a weekday
+   *
+   * @remarks The property is defined only dates during Christmas Time.
+   */
+  isNativityOfTheLordOnWeekday?: boolean;
```

Comment:
You might want to change the property name.

Reply:
you could just make it private or available through a getter, in which case it shouldn't be serialized.

Reply:
this is pretty much what i was trying to avoid, you can't really not expose it if the type says you do... i don't think we can avoid having it either on the final object if the type dictates it will be defined during Christmas Time.

Reply:
imo we should expose the Seasons here again and use those to determine the psalter week rather than determining a boolean in the calendar instance to then be used in the liturgical day. it would make sense to pass metadata for allowing the logic of what the day is precisely so the day can handle all the logic there.

Reply:
pushing something shortly.

Reply:
pushed